plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -81,11 +102,46 @@ jQuery( function( $ ) {
 
 				return false;
 			}
+		} )
+		.on( 'keyup', function() {
+			var $activeElement = $( document.activeElement );
+			var sliderKeyupBlockers = [ '.stars', '.tabs', '.wc-tabs'];
+			var $closestBlocker = $activeElement.closest( sliderKeyupBlockers.join( ', ' ) );
+
+			if ( $closestBlocker.length ) {
+				// Prevent keyup events from being triggered on the flexslider when the focus is on the stars or tabs.
+				productGalleryElement.data('flexslider').animating = true;
+				return;
+			}
+
+			productGalleryElement.data('flexslider').animating = false;
+		})
```

Comment:
When the focus is on an element inside the `.stars` or `.tabs` wrappers, we want the arrow keys to move across their nested elements and not to move the product slider.
Looking into the [Flexslider plugin](https://github.com/woocommerce/FlexSlider/blob/master/jquery.flexslider.js#L109), I realized that setting `animating` to `true` would do the trick.